### PR TITLE
Seo-171884-Page-with-redirect-fix-2

### PR DESCRIPTION
**Pull request by Thahira-Banu**
Hi @Aishwarya-Ganesan 
I had made changes in page with redirect links by replacing redirecting urls.

Thanks & Regards

### CODE REVIEW

File: uwp/Charts/Series.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ chart.Series.Add(series2);
 
 {% endtabs %}
 
-![Colum SegmentSpacing support in UWP Chart](Series_images/SegmentSpacing.png)
+![Colum SegmentSpacing support in UWP Chart](series_images/segmentspacing.png)
```
Comment (MallikaRK) [inline uwp/Charts/Series.md:141]: You make the UWP/Chart/Series.md file changes in the below PR too.
https://github.com/syncfusion-content/uwp-docs/pull/371/files
Which one is Final?

Comment (MallikaRK) [reply uwp/Charts/Series.md:141]: Seems both PR's are duplicate. Pleas check and share the Final PR.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Comment by Thahira-Banu** [comment]
I have made changes in multiple PR's , so closed it.

**Pull request closed without merging**